src/josepy/jwk.py

```diff
@@ -248,7 +225,7 @@ def fields_from_json(cls, jobj):
 
             key = rsa.RSAPrivateNumbers(
                 p, q, d, dp, dq, qi, public_numbers).private_key(
-                    default_backend())
```

Comment:
What's the reason for this change?

Reply:
My IDE must have auto-formatted this. I will revert this change.

Reply:
Fixed by 2c3cabe3